userspace/libsinsp/container_engine/sinsp_container_type.h

```diff
@@ -20,16 +20,17 @@ limitations under the License.
 
 enum sinsp_container_type
 {
-	CT_DOCKER = 0,
-	CT_LXC = 1,
-	CT_LIBVIRT_LXC = 2,
-	CT_MESOS = 3,
-	CT_RKT = 4,
-	CT_CUSTOM = 5,
-	CT_CRI = 6,
-	CT_CONTAINERD = 7,
-	CT_CRIO = 8,
-	CT_BPM = 9,
-	CT_STATIC = 10,
-	CT_PODMAN = 11,
+	CT_UNKNOWN = 0,
```

Comment:
Do you see any issue with changing the 0 value of this enum? If this is serialized somewhere it's going to be a problem and I can set the unknown value to something like `0xffff`. It does not look like it's serialized to scap but maybe I missed something ( cc @gnosek )

Reply:
Hmm doesn't this end up in scap files? https://github.com/falcosecurity/libs/blob/e54666d1f1c79fffbb91a22fad9ce38e3ddc379e/userspace/libsinsp/container.cpp#L172

Reply:
Yep, `0xffff` it is then